src/main.js

```diff
@@ -160,6 +150,7 @@ let load = loader({
           signatureValidator,
           sentryManager,
           statsum:            cfg.app.statsum,
+          monitor:            monitor.prefix('api-context'),
```

Comment:
Is this 'api-context' pattern repeated in some other project? I don't think this makes a lot of sense. Is it generally added to the context elsewhere at all?